pkg/controller/subnet.go

```diff
@@ -429,13 +429,14 @@ func (c Controller) patchSubnetStatus(subnet *kubeovnv1.Subnet, reason string, e
 
 func (c *Controller) handleAddOrUpdateSubnet(key string) error {
 	var err error
-	subnet, err := c.subnetsLister.Get(key)
+	orisubnet, err := c.subnetsLister.Get(key)
```

Comment:
here is used for finalizer, the actual used variable is downside, so it is ok not to modify here.

Reply:
right. modified